web/html/src/branding/css/uyuni-typography.less

```diff
@@ -0,0 +1,18 @@
+@font-face {
+  font-family: 'Roboto Light';
+  font-style: normal;
+  font-weight: 300;
+  //src: url('Roboto/Roboto-Light.eot');
+  // TODO: Fix
+  // src: local('Roboto Light'), local('RobotoLight'), url('../fonts/Roboto/Roboto-Light.ttf') format('truetype');
```

Comment:
A question for @ncounter , these can't currently resolve correctly with Webpack because they come from the frontend libs package. Is there a reason to not have this font directly in the assets similar to other fonts?

Reply:
Absolutely no reason at all to not move it. And there are others they could follow this train.
The historical reason for the `frontend-libs` was decided (I guess) at the beginning of the project: keep in github only our maintained and implemented code, everything from the outside jumps in at build time in ibs/obs. I believe this is an outdated statement and we already went for different solutions in different cases so, :+1: to move at least the trivial one.